tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -304,13 +308,72 @@ def _get_effective_padding_node_input(stride, padding,
   return stride * effective_padding_output + padding
 
 
-def compute_receptive_field_from_graph_def(graph_def, input_node, output_node):
+class ReceptiveField:
+  """
+  Receptive field of a convolutioanl neural network.
+
+  Args:
+    size: Receptive field size.
+    stride: Effective stride.
+    padding: Effective padding.
+  """
+  def __init__(self, size, stride, padding):
+    self.size = np.asarray(size)
+    self.stride = np.asarray(stride)
+    self.padding = np.asarray(padding)
+
+  def compute_input_coordinates(self, x, axis=None):
```

Comment:
what about renaming this to "compute_input_center_coordinates"?
I believe the current form is ambiguous, since "coordinates" could correspond to the "bounding box" coordinates of the receptive field region (ie, top-left and bottom-right corners).